### DISCUSSION TITLE
SNMP, sampling rate, template errors

### DISCUSSION THREAD

**Discussion by dubefab** [akvorado/akvorado#515] [Q&A]
I am seeing traffic come to akvorado but after over 12h those errors keep piling up and nothing shows on the UI.

user@akvorado:~$ curl -s http://127.0.0.1:8080/api/v0/inlet/metrics | grep '^akvorado\_inlet' | grep \_errors
akvorado_inlet_core_flows_errors{error="SNMP cache miss",exporter="192.168.100.1"} 438520
akvorado_inlet_core_flows_errors{error="sampling rate missing",exporter="192.168.100.1"} 438520
akvorado_inlet_flow_decoder_netflow_errors_count{error="template not found",exporter="192.168.100.1"} 120

**Comment by dubefab** [comment]
How do I add this option to a cisco 5506?
option sampler-table timeout 10

**Comment by vincentbernat** [comment]
You can specify the sampling rate in the configuration file if you want. For the SNMP cache miss, the more likely cause is that you didn't configure SNMP correctly.
